phonon/update.py

```diff
@@ -236,3 +230,117 @@ def merge(self, update):
         """
         raise NotImplemented("You must define a merge method that merges it's\
             argument with this object.")
+
+class Update(BaseUpdate):
+
+    def _BaseUpdate__merge(self):
+        pickled = self._BaseUpdate__process.client.get(self.resource_id)
```

Comment:
Also not officially supported. Only override public methods. If you need to override private methods you should define a new base class.